packages/grid_client/scripts/applications/casberlabs.ts

```diff
@@ -0,0 +1,142 @@
+import { FilterOptions, GatewayNameModel, MachinesModel } from "../../src";
+import { config, getClient } from "../client_loader";
+import { log } from "../utils";
+
+async function pingNodes(client, nodes) {
+  for (const node of nodes) {
+    try {
+      await client.zos.pingNode({ nodeId: node.nodeId });
+      return node.nodeId;
+    } catch (error) {
+      console.log("node " + node.nodeId + " is not responding, trying different node.");
+    }
+  }
+  throw new Error("No avaiable nodes");
+}
+
+async function deploy(client, vms) {
+  const res = await client.machines.deploy(vms);
+  log("================= Deploying VM =================");
+  log(res);
+  log("================= Deploying VM =================");
+}
+
+async function deployGW(client, gw) {
+  const res = await client.gateway.deploy_name(gw);
+  log("================= Deploying name gateway =================");
+  log(res);
+  log("================= Deploying name gateway =================");
+}
+
+async function getDeployment(client, vms) {
+  const res = await client.machines.getObj(vms);
+  log("================= Getting deployment information =================");
+  log(res);
+  log("================= Getting deployment information =================");
+  return res;
+}
+
+async function getDeploymentGW(client, gw) {
+  const res = await client.gateway.getObj(gw);
+  log("================= Getting deployment information =================");
+  log(res);
+  log("================= Getting deployment information =================");
+  return res;
+}
```

Comment:
as you did in `cancel` you delete both in a single function, you can do the same here and in deploy, you can pass a param that tells you what unit you want to deploy/get